doc/api/modules.md

```diff
@@ -165,7 +165,7 @@ require(X) from module at path Y
 6. THROW "not found"
 
 LOAD_AS_FILE(X)
-1. If X is a file, load X as JavaScript text.  STOP
+1. If X is a file, load X as its file extension format.  STOP
```

Comment:
currently this always loads as CJS in `require()`, right?

Reply:
This is actually a spec correction to CommonJS I made while I was here in that if `X = 'file.json'`, it certainly won't be loaded as JavaScript but rather as JSON, when doing `require('./file.json')`. Strictly speaking by the spec right now, it is saying to load explicit JSON extensions as JS :)